### ISSUE TITLE
Abandoned?

### ISSUE THREAD

**Issue by ghost** [kolber/stacey#131]
I really like stacey, also its not mobile friendly. But it seems that its now abandoned, right?

**Comment by Freundschaft** [comment]
doesnt mobile friendliness depend on the template rather than stacey?

**Comment by ghost** [comment]
Yes. But the three themes arent mobile friendly. and no new version comes up of stacey. sadly.

**Comment by Freundschaft** [comment]
I'm using stacey for quite a while now and I'm quite happy with it, are there any urgent things that need to be fixed? maybe someone can update the default themes to be mobile friendly.

**Comment by pbatts** [comment]
Abandoned? I think not. The last communication I saw from Anthony Kolber was on 8/26/14. Also, there are still plenty of users, like us, out there.
Are you all using v3.0.0? I have been working on updating template2 and template3 to work with v3.0.0.
Unfortunately, the existing, stock, templates are not responsive.
I don't use any of the stock templates. I only use custom templates that I have created and made responsive.

**Comment by Freundschaft** [comment]
I'm still on 2.x, the offical website still lists that version as the most recent version, 
http://www.staceyapp.com/ 

is there something like a stacey showcase? or template collection?

**Comment by pbatts** [comment]
Unfortunately that website has not been kept up to date. I'm not sure why Kolber has not pushed the newest v3.0.0 and associated documentation to that site. Or maybe even better, create a new site for v3.0.0.

It can be very confusing.

The latest version is 3.0.0. The template language is Twig. The template syntax is different from v2.3.0.

I have been using v3.0.0, in production, since it was released.

The best source for v3.0.0 and related information is https://github.com/kolber/stacey
Documentation for the v3.0.0 can be found at https://github.com/kolber/stacey/wiki

**Comment by o-l-e** [comment]
i don't think Stacey is abandoned, as @pbatts says there are plenty of us using Stacey.
Would be great if @kolber could maybe let us know about plans for Stacey in the future :)
I really do love using Stacey, it has so many great qualities, and is very easy to work with responsive templates.

Ps. don't mean to push Kolber here, take it more as compliments for fantastic work.

**Comment by ghost** [comment]
A thing can be abandoned even its in use. Games can be abandoned just when the publisher "forget" them. So I hope @kolber kolber didnt want to forget his work.

I am acutally using Stacey 3.00 (the fact that the homepage is outdated sucks really hard) with the theme from https://github.com/rubrodapa/Staceytrap . 

Sadly php isnt my language, I cant work on it by myself. May someone can publish the normal themes with mobile support and look into the code for smaller files and better commenting and so.

EDIT: I wrote him and will publish any news if he answer me.

**Comment by o-l-e** [comment]
@visurox i am sure @kolber is trying to do what he can. After all Stacey is free, and he has put an awful lot of good work into the project. Stacey is after all a CMS, and i personally think it's great that it focuses on being a CMS not piled up and bloated with frameworks for css and this and that. 

A lot has been happening in frontend development in recent years, and the fact that Stacey is as minimal as it is, makes i easier for people to get started with their own templates without spending most of the time figuring out what and where all the bits and pieces are (like WP or similar).

You should see https://getsatisfaction.com/stacey if you are looking for ready templates, i have seen some there. Like this one: https://getsatisfaction.com/stacey/topics/new_template_with_fluid_dynamic_layout / http://www.yconst.com/web/assemblage/

If you really like Stacey, learn to make templates by using forums, stackoverflow etc, or maybe donate to the project ;)

**Comment by pbatts** [comment]
@o-l-e Totally agree.
@visurox Perhaps I am misunderstanding your concerns. One of the wonderful things about Stacey is you do not need to know <code>.php</code> to fully utilize what Stacey has to offer as a CMS. If you are familiar with <code>.html</code> and <code>.css</code> you are well on your way. The only thing remaining is to learn and understand Stacey's templates. Soon you will be creating and customizing your own templates for your Stacey powered website.

Also, https://github.com/rubrodapa/Staceytrap is a Bootstrap template created by @rubrodapa and is not the original Stacey 3.0.0. So, I don't understand your comment "(the fact that the homepage is outdated sucks really hard)". Please explain.

**Comment by Freundschaft** [comment]
I think he is referring to the following homepage: http://staceyapp.com/

**Comment by pbatts** [comment]
Ahhh. Thanks @Freundschaft 

It is important to note, for anyone new to Stacey that there are two versions. v2.3.0 and v3.0.0. With the two different versions being spread out over multiple websites, it can be confusing. My suggestion for anyone just starting out with Stacey would be to use the version v3.0.0. That is the latest and greatest. I would encourage anyone still using v2.x.x to make the jump to v3.0.0, though it will require a considerable amount of work.

The best source for v2.3.0 information is http://www.staceyapp.com/ and https://getsatisfaction.com/stacey as mentioned by @Freundschaft and @o-l-e above.

The best source for v3.0.0 information is https://github.com/kolber/stacey and https://github.com/kolber/stacey/wiki

**Comment by pbatts** [comment]
@visurox @Freundschaft Regarding 'mobile friendly', meaning a Responsive design, it is not likely nor would it be prudent to put energy into updating the templates for any of the v2.x.x versions.

Responsive templates for v3.0.0 are on my 'to do' list. I hope to be able to start work on these fairly soon.

**Comment by ghost** [comment]
Sound great and I would donate if I find a button or link for that! :)

Its just confsing for me that the official site says the last was 2.3 and the official github has a much better version. But if you trust the homepage (seek a download button and thats it) you will never found out that there are a 3.0 version. Staceytrap is just a mobile template I using for stacey, I didnt want to say more with this. 

So it seems that I should look into better css knowledge to make my own.

Stacey was exactly what I seek (after a long time for searching). Yes, because of no bloatshit. 
I try many of so called CMS or flat file systems, most of them are really ugly, unsupported, abandoned or need node.js (like Ghost). The Pulse CMS, nibbleblog and Yellow (Datenstrom.se) are the only I found who are ok with me. But Stacey are... great!

**Comment by o-l-e** [comment]
on the front page bottom right http://staceyapp.com there is a donation link ;)
Keep it up, it's great fun when you get going.

**Comment by ghost** [comment]
Your right, thank you. Will spend some bucks, I should made this long time before...

**Comment by Freundschaft** [comment]
also stacey is an opensource project now so I guess everyone can contribute :)

**Comment by mjau-mjau** [comment]
I would like to chime in on this conversation. In my opinion, I would say Stacey is **almost** abandoned by the original creator ... Otherwise, the least that could be done, would be to update the staceyapp.com website to reflect version 3.0. I wouldn't blame the creator though ... I assume his private- and professional commitments have made it difficult to follow up on an open source project.

Having said that, I would not say that Stacey is outdated in any way. About a year ago, I spent several weeks reviewing and testing small PHP frameworks for our project, and Stacey came through on top. After working with Stacey for a year, creating a modern, resonsive html5 ajax frontend and adding extensions (twig, asset types etc), I can easily conclude that Stacey is (in 2014) a very smart application for what it is made out to do.

I already have an extended Stacey, including an early version of a control panel. I was considering creating a new branch for Stacey, or even re-developing a new ('inspired by Stacey') project wrapped in a framework like Laravel (assisted dev), and extending documentation and community in a separate website. I cannot entirely commit to that idea yet, but I am dwelling on it ...

**Comment by domenicomazza** [comment]
Agree, and while the updates aren't rolling in, there's nothing to say Stacey itself is old in any way—it's perfect for many web projects that don't really need a server database and simpler than other flat-file CMS. The only real thing it is missing is the 'back-end' content management, not that it doesn't exist, it's just not documented here on GitHub http://www.kolber.info/#/project6 ... once that's here, the term 'abandoned' can just be 'finished' :)

**Comment by moebiusfrost** [comment]
Are you guys still using Stacey? I'm still considering my choice for my design portfolioo. I heard that Grav CMS, Pico CMS or Kirby are an alternative. What do you guys think?

**Comment by mjau-mjau** [comment]
I would go for Grav, as it is a natural successor to Stacey, and basically what Stacey "should have been". It uses the same (near-identical) smart techniques as Stacey, but built around a more extensive and modern PHP architecture to meet standards in 2015. Furthermore, it has extensive documentation, a growing user-base and is seemingly under steady development by the folks behind RocketTheme.

I haven't tried Grav yet, because I can't currently justify the time to move a large project from Stacey to Grav. I will be moving to Grav as soon as I get the opportunity.

**Comment by pbatts** [comment]
I am still using Stacey. I have been using it for several years. I have looked at Pico and Kirby but stuck with Stacey.

I just spent three weeks working with Grav. I am sticking with Stacey. It suits my needs perfectly. Grav is similar to Stacey but different in a lot of ways. It has many features but with those features comes complexity. Grav is anything but simple. A good deal of work will be required to port your Stacey site to Grav.

I would add that if you are just starting out building your site and have never used any of the above mentioned CMS's, Grav is the logical modern choice.

**Comment by o-l-e** [comment]
I have spent countless hours with Stacey, and i am still using it for small projects and prototypes. Stacey has so many nice features, and though there are several other CMS out there, i haven't had the time to switch to any other yet. It's a bit sad that there is no more development, i still really like it.

**Comment by pbatts** [comment]
Yes, it is too bad there is no more development. My biggest complaint about Stacey is the lack of a "Plugin" type architecture and not being able to easily upgrade the Twig version.

**Comment by rfpuyana** [comment]
Any news if Stacey is gonna be kept updated? Any ideas of other light CMS alternatives?

**Comment by skyballs** [comment]
I've been using Stacey for a few years & it's pretty good for what I need. A couple months ago my host contract expired (opps), & after reinstating my site after payment, the site navigation is now broken. From the homepage, clicking on any link takes you to the correct url but what you see is a copy of the homepage with broken image links, not the page which should be there. I went to reupload all the files but they were all restored to their correct locations by the host, so I don't know if that would do anything. Has anyone encountered this or anything similar with Stacey?

**Comment by mjau-mjau** [comment]
Did you copy in the Stacey **.htaccess** file? What you describe sounds typically like rewrite rules not working, and rewrite rules are in the .htaccess file. This file is often hidden (system file) and is easily overlooked when copying or moving files.

**Comment by o-l-e** [comment]
@skyballs i had a similar issue when a host upgraded the php. If this is the case with your host, you can try this: https://github.com/kolber/stacey/issues/161

**Comment by desbest** [comment]
I also am interested in knowing whether Stacey is abandoned or not. I use it for my portfolio website. It's a good CMS, so it needs to be maintained.

**Comment by pbatts** [comment]
Eight years later and I'm still using Stacey. Anyone else?

I have been using Stacey 3.0.0 for nine years, now, to run a commercial website with 1500+ product items. I don't run Stacey on the server, I compile the site into static html and upload that to the server. It is safe, secure and lightning fast.

My only complaint, still to this day, is it runs on Twig v1.4. I would be willing to pay if there is someone that can update the Twig version in Stacey.

Some of the doc links above are dead. Luckily, I managed to save all of that documentation before it disappeared. Sorry for not posting this sooner. That info can be found at https://www.guitdev.com

**Comment by pbatts** [comment]
> I also am interested in knowing whether Stacey is abandoned or not. I use it for my portfolio website. It's a good CMS, so it needs to be maintained.

Abandoned by the developer but not by the community.

**Comment by desbest** [comment]
Let's not forget that Stacey is the FIRST ever flat-file CMS in the world that used structured content data formats. Without Stacey, it's derivatives like Kirby, Grav, Automod and Perch wouldn't exist.

Before Stacey came along, it was more about WYSIWYG with content blocks, for a site builder like [Zimplit](https://www.youtube.com/watch?v=XM7Pyi4blgg) and [Mojomotor](https://www.youtube.com/watch?v=-Rsw3gfkx2Q) or simple things like [Website Baker](https://www.opensourcecms.com/website-baker/).

**Comment by pbatts** [comment]
> Let's not forget that Stacey is the FIRST ever flat-file CMS in the world that used structured content data formats. Without Stacey, it's derivatives like Kirby, Grav, Automod and Perch wouldn't exist.
> 
> Before Stacey came along, it was more about WYSIWYG with content blocks, for a site builder like [Zimplit](https://www.youtube.com/watch?v=XM7Pyi4blgg) and [Mojomotor](https://www.youtube.com/watch?v=-Rsw3gfkx2Q) or simple things like [Website Baker](https://www.opensourcecms.com/website-baker/).

Thank you for that.

**Comment by rfpuyana** [comment]
Stacey is brilliant. I hope it comes back. I am not a developer so hard for me to contribute.
